### DISCUSSION TITLE
Multi-Select User Access

### DISCUSSION THREAD

**Discussion by Kuuchuu** [Budibase/budibase#4893] [Ideas]
I know that the groups feature is a paid feature, but would it potentially be possible to get a way to a select multiple apps and grant a user access for all selected apps at once? Like a checkbox next to each app in the app list on the user's configuration roles area so I can set "Basic" access for the user for 8 different apps? It's a major pain adding in 40 users and granting each one basic access to each app.

**Comment by mjashanks** [comment]
Thats a good suggestion, thanks! In the meantime, you could potentially write a script for this using our REST API, if you're willing to write a bit of code.
